studio/components/SalaryCsvInput/SalariesInput.tsx

```diff
@@ -0,0 +1,146 @@
+import { set, StringInputProps } from "sanity";
+import { Box, Grid, Inline, Stack, Text, useTheme, useToast } from "@sanity/ui";
+import { ChangeEvent } from "react";
+import styles from "./salariesInput.module.css";
+import { UploadIcon } from "@sanity/icons";
+import {
+  salariesAsStoredString,
+  salariesFromCsvString,
+  salariesFromStoredString,
+} from "./salariesParseUtils";
+import { SalariesParseErrorsToastDescription } from "./SalariesParseErrorsToastDescription";
+import SalaryNumberInput from "./SalaryNumberInput";
+
+const UPLOAD_CSV_INPUT_ID = "upload-csv-input";
+
+export const SalariesInput = (props: StringInputProps) => {
+  const toast = useToast();
+  const theme = useTheme();
```

Comment:
Do we need this? Maybe we could find a way to do this in module.css

Reply:
This accesses the theme (light/dark/system) that the user has chosen in Sanity Studio. I don't see how this can be done in `module.css`, but I am open to suggestions 🤗

Reply:
Actually just found that we can use `[data-scheme="dark"]` as a CSS selector 💡

Reply:
Ooh nice — maybe we can use the data-scheme in it's own PR, because the other custom components are in need of this!

Reply:
So should I just keep `useTheme` here for now?

Reply:
Wouldn't it be better to change it now and replace the other custom components in a separate PR later?

Reply:
Sure, will fix :+1: